src/commands/asset/edit.rb

```diff
@@ -6,6 +6,8 @@ module Asset
       class Edit < CommandHelpers::RecordEditor
         private
 
+        include CommandHelpers::HasAssetConcern
```

Comment:
I don't think the `Edit` command needs to include `HasAssetConcern` explicitly. It should already be included by `RecordEditor`